GameLogic/entities/pieces/PieceConstants.cs

```diff
@@ -15,7 +15,7 @@ static class PieceConstants
         public static readonly ISet<int> InvalidPawnRanks = new HashSet<int> { Y._0, Y._7 };
 
         public static readonly ISet<PieceType> PossiblePromotionPieceTypes =
-            new HashSet<PieceType>{PieceType.Knight, PieceType.Bishop, PieceType.Rook, PieceType.Queen};
+            new HashSet<PieceType> { PieceType.Knight, PieceType.Bishop, PieceType.Rook, PieceType.Queen };
```

Comment:
Are spaces correct? As I get it, you planned to undo adding such spaces.

Reply:
These spaces are not the same as I showed recently